sty/statements/statements.dtx

```diff
@@ -1364,10 +1364,8 @@ DefConstructor('\inlinedef OptionalKeyVals:omtext {}', sub {
    }}
  my @symbols = @{$props{defs} || []};
  #Prepare for symbol insertion -insert before the parent of the closest ancestor CMP element
-  my $original_node = $document->getNode;
- my $xc = XML::LibXML::XPathContext->new( $original_node );
- $xc->registerNs('omdoc', 'http://omdoc.org/ns');
```

Comment:
I used the accessor: 
http://dlmf.nist.gov/LaTeXML/manual/coremodules/LaTeXML_Core_Document.html
It seems to be working without registering the namespace and I have tested it in `omtext`